react-native/react/menubar/index.render.desktop.js

```diff
@@ -28,6 +28,25 @@ const Header = props => {
   const showUser: () => void = props.showUser
 
   const version = __VERSION__ // eslint-disable-line no-undef
+  return (
+    <div style={styles.header}>
+      <FontIcon className={`fa fa-folder`} style={{...styles.icons, fontSize: 17, marginRight: 10, cursor: 'default'}} color={globalColors.grey2} hoverColor={globalColors.grey1} onClick={openKBFS} />
```

Comment:
We should just update the common Icon class to allow the hover to be passed in and not use the raw FontIcon from material-ui and all these inlined styles